### ISSUE TITLE
[Profile, Trackers] Fix 'classes' of trackers to roll up

### ISSUE THREAD

**Issue by johnsturgeon** [johnsturgeon/apex-legends-tracker#97]
https://discord.com/channels/142164363703222272/542110550575022090/841395759613345833

So.. in a nutshell here it is:
* Profile Page Trackers should match your trackers exactly 1:1
  Meaning: Damage is literally just the 'damage' tracker
* Daily stats / games is calculated a bit differently.  There I have aggregated some trackers to get a better picture.  So, for instance 'specialEvent_damage' is used if 'damage' can't be found
So the fix we did yesterday for you @Basicide [XB1/PC] was to add 'soiree_XXX` to my tracker aggregation logic for game / game totals
Bottom line:
* Right now, I'd consider "Behaves Correctly" .. BUT... I think it would be nice if we could have formal 'tracker categories' where .. for instance the logic would go something like this:
For each legend, check to see if they have 'damage'.  If no 'damage', then check for 'specialEvent' and 'soiree'.
if those don't exist, then loop through all the season_n_damage trackers and add them up
